README.md

```diff
@@ -1 +1,59 @@
-# angular-manual-transclusion-readme
\ No newline at end of file
+# Manual and isolate transclusion
+
+## Overview
+
+Now that we know what transclusion does, we can look into more advanced methods of implementing it. Let's take a look how to transclude our content manually.
+
+## Objectives
+
+- Use the transclude function inside the link function
+
+## transclude()
+
```

Comment:
Can you set this up by explaining why we might need this? What's an example of a situation where you'd need to use manual transclusion?